.github/dependabot.yml

```diff
@@ -60,7 +60,6 @@ updates:
       - '/docs'
       - '/graders/c'
       - '/graders/python'
-      - '/images/plbase'
```

Comment:
Do you need to add `/` explicitly? Or is that automatic? I ask because for both `docker` and `npm` above, we explicitly add `/`.

Reply:
Fixed in [0ce52bf](https://github.com/PrairieLearn/PrairieLearn/pull/12293/commits/0ce52bf09914e9455bd038e43f70b1c9c6c02cb4)